readme.md

```diff
@@ -119,14 +112,30 @@ Although this [must be run manually](#linting-your-changes), developers may wish
 
 
 The following dependencies aren't needed by most people, and are not included in Git submodules:
-
-* To generate developer documentation: [Sphinx](http://sphinx-doc.org/), version 3.4.1
 * To generate developer documentation for nvdaHelper: [Doxygen Windows installer](http://www.doxygen.nl/download.html), version 1.8.15:
 * When you are using Visual Studio Code as your integrated development environment of preference, you can make use of our [prepopulated workspace configuration](https://github.com/nvaccess/vscode-nvda/) for [Visual Studio Code](https://code.visualstudio.com/).
 	While this VSCode project is not included as a submodule in the NVDA repository, you can easily check out the workspace configuration in your repository by executing the following from the root of the repository.
 
 	```git clone https://github.com/nvaccess/vscode-nvda.git .vscode```
 
+### Python dependencies
+NVDA and its build system depend on an extensive list of Python packages. They are all listed with their specific versions in a requirements.txt file in the root of this repository. This file can be used by the Pip Python package manager to install all required packages.
+These dependencies can be installed with the command:
+```
+py -m pip install -r requirements.txt
+```
+However, it is strongly recommended that you create a Python virtual environment for this repository, and activate it before installing required python packages and or building or running NVDA. This avoids any Python package version conflicts with packages you may have installed system-wide.
+To create a Python virtual environment in the .venv directory:
+```
+py -m venv .venv
+```
+To activate the virtual environment:
+```
+.venv\scripts\activate.bat
+```
```

Comment:
Actually, I would switch the paragraphs around and simply instruct users to first create a virtual environment and then run the pip command. Since this is what AppVeyor will also do, and it is good practice anyway, why bother with howevers and maybes? Just my 2 Cents.

Reply:
I agree. I think we should move all the "doing a local build" stuff into a dedicated file or at least section. It should be ordered so it can be followed step by step.